src/bluefinClient.ts

```diff
@@ -2305,6 +2308,18 @@ export class BluefinClient {
     }
   };
 
+  /**
+   * @description
+   * claim all vaults in batch contract call
+   * @param batch array containing SignaturePayload and user's signature
+   * @returns ResponseSchema
+   */
+  calimFromVaultBatch = async (
```

Comment:
This is a public repo, we need to hold our code quality here to a very high standard. Please fix the typo here